asserts/database.go

```diff
@@ -207,6 +218,15 @@ func OpenDatabase(cfg *DatabaseConfig) (*Database, error) {
 		}
 	}
 
+	otherPredefinedBackstore := NewMemoryBackstore()
+
+	for _, a := range cfg.OtherPredefined {
+		err := otherPredefinedBackstore.Put(a.Type(), a)
```

Comment:
Should we allow the same assertion to be in the trusted *and* predefined backstore? It may cause duplicates later. However, I'm happy with an answer, "it's a configuration error, don't do that" :).